source/preferences/manage-your-web-notifications.rst

```diff
@@ -0,0 +1,77 @@
+Manage your web notifications
+==============================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. |chrome-mention-badge| image:: ../images/chrome-mention-badge.png
+  :alt: An example of a Chrome web browser mention badge indicator.
+
+.. |chrome-activity-badge| image:: ../images/chrome-activity-badge.png
+  :alt: An example of a Chrome web browser activity badge indicator.
+
+Enable notifications
+--------------------
+
+From Mattermost v9.10, Mattermost prompts you to enable notifications in your web browser.
+
+.. image:: ../images/enable-notifications.png
+  :alt: From Mattermost v9.10, you're prompted to enable notifications.
+
+- When you select **Enable notifications**, you won't be asked again. You'll start receiving notifications in your web browser for all Mattermost activity with `badges <#badge-based-notifications>`__ and `sounds <#notification-sounds>`__. See the section below on `customizing your notifications <#customize-your-notifications>`__ based on how you prefer to be notified about Mattermost activity in a web browser.
+- If you dismiss this prompt, you won't receive Mattermost notifications in the web browser, and you'll prompted again the next time you open Mattermost in a web browser.
+
+Badge-based notifications
+-------------------------
+
+In a web browser, Mattermost icons display the following types of badges:
+
+- Numbered badges for unread :ref:`direct <collaborate/channel-types:direct messages>` :ref:`group <collaborate/channel-types:group messages>` messages, :doc:`@mentions </preferences/manage-your-mentions-keywords-notifications>`, and :doc:`keywords </preferences/manage-your-mentions-keywords-notifications>` you're actively watching.
+
+ - A red dot badge means an unread @mention or keyword. |chrome-mention-badge|
+
+ - A black dot badge means an unread direct or group message. |chrome-activity-badge|
```

Comment:
The black dot is for unread activity in channels your are following but not for direct or group message that is also the red dot https://github.com/mattermost/docs/pull/7378#discussion_r1765582066